addons/website_sale/controllers/main.py

```diff
@@ -872,6 +872,7 @@ def payment_get_status(self, sale_order_id, **post):
                 values.update({
                     'tx_ids': True,
                     'state': state,
+                    'order': order,
```

Comment:
@tde: redirect  into that order's particular payment_acquirer_id. So user dont have to find that wire transfer from list. it will redirect to the chosen wire transfer form.
Thanks